test/scrabble_test.rb

```diff
@@ -5,8 +5,38 @@
 
 class ScrabbleTest < Minitest::Test
```

Comment:
Don't forget to test your helper methods!  And I would have expected to see a test like this `assert_equal 8, game.score('hello')`